runners/flink/runner/src/main/java/org/apache/beam/runners/flink/FlinkStreamingPipelineTranslator.java

```diff
@@ -227,4 +246,73 @@ private ReflectiveOneToOneOverrideFactory(
     }
   }
 
+  /**
+   * A {@link PTransformOverrideFactory} that overrides single-output {@link ParDo} to implement
+   * it in terms of multi-output {@link ParDo}.
```

Comment:
Perhaps link here to https://issues.apache.org/jira/browse/BEAM-1709 ? This code will go away once https://github.com/apache/beam/pull/2170 is un-reverted.